pkg/apis/externaldns/types.go

```diff
@@ -290,6 +292,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("exclude-domains", "Exclude subdomains (optional)").Default("").StringsVar(&cfg.ExcludeDomains)
 	app.Flag("zone-id-filter", "Filter target zones by hosted zone id; specify multiple times for multiple zones (optional)").Default("").StringsVar(&cfg.ZoneIDFilter)
 	app.Flag("google-project", "When using the Google provider, current project is auto-detected, when running on GCP. Specify other project with this. Must be specified when running outside GCP.").Default(defaultConfig.GoogleProject).StringVar(&cfg.GoogleProject)
+	app.Flag("google-batch-change-size", "When using the Google provider, set the maximum number of changes that will be applied in each batch.").Default(strconv.Itoa(defaultConfig.GoogleBatchChangeSize)).IntVar(&cfg.GoogleBatchChangeSize)
```

Comment:
Do you also want to add an interval for that? We have it for AWS, so I don't know it make sense to provide it for Google as well.

Reply:
I think that it makes sense to add an interval like it is done for AWS. I will work on it and update the PR.